### PR TITLE
[splib3] adds serial port utils

### PR DESCRIPTION
**Pull request by EulalieCoevoet**

### CODE REVIEW

File: python3/src/splib3/interface/serialport.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    if not ports:
+        Sofa.msg_error("stlib3.interface.serialport",
+                       "No serial port found with " + method + " = " + entry)
+        return
+
+    if len(ports) > 1:
+        Sofa.msg_warning("stlib3.interface.serialport",
+                         "Multiple port found with " + method + " = " + entry + ". Using the first.")
+
+    print("---------------------------------------")
```
Comment (damienmarchal) [inline python3/src/splib3/interface/serialport.py]: print  ?

Comment (EulalieCoevoet) [reply python3/src/splib3/interface/serialport.py]: I think it's interesting to print the info of the device it found. We can discuss that. 
The print is just to pack the info. I did the same as SofaPython3 does when checking for SOFA_ROOT. 

image

Comment (EulalieCoevoet) [reply python3/src/splib3/interface/serialport.py]: I've removed it.

**Comment by EulalieCoevoet** [comment]
@damienmarchal I removed the print, should we merge? Or do you see another issue?

**Comment by damienmarchal** [comment]
@EulalieCoevoet  looks ok to me.

**Pull request merged**